en/test_and_ci/integration_testing.md

```diff
@@ -21,8 +23,8 @@ cd <PX4-Autopilot_clone>
 make px4_sitl_default sitl_gazebo
 make <test_target>
 ```
-test_target is one of the Makefile targets. The available ones are:
-*tests_mission*, *tests_mission_coverage*, *tests_offboard* and *tests_avoidance*.
+
+`test_target` is a makefile targets from the set: *tests_mission*, *tests_mission_coverage*, *tests_offboard* and *tests_avoidance*.
```

Comment:
What tests are run in each? Is there a way for someone to work out what our tests cover and what they don't